.github/workflows/bix_docker.yml

```diff
@@ -11,11 +11,6 @@ name: BIX Docker
   pull_request:
     branches:
       - master
-    types:
-      - opened
-      - synchronize
-      - labeled
-      - reopened
```

Comment:
opened, synchronized and reopened are the default types. Labeled is added so we're able to add the label. I guess it's not really any additional work to just manually re-run the workflow after we add the label to our own PRs. It would obviate the need for the concurrency groups below.

The other thing we could try is to remove the "default" labels from the dependabot.yml file. The documentation makes it sound like if you specify any labels there, it will not add the default labels but looking at the jobs that get kicked off, that doesn't seem accurate?

Reply:
> opened, synchronized and reopened are the default types. 

While they are the defaults there's something different about how they are types are de-duplicated when not listed. I'm reaching here.

>  just manually re-run the workflow after we add the label to our own PRs.

I added that, thanks for the call out.